app/assets/javascripts/vendor/lib/google/maps.js.erb

```diff
@@ -1,419 +1,417 @@
                 }
                 wx = wx - 0;     ////  DUMMY}
               }
-              wx = wx - 0;     ////  DUMMY}
-            }
-          }
-          //if (bounds.anti || bounds.prime) {
-          //  //if ((wm > 179) && (wp > 179)) {     // check for prime meridian and anti-meridian case
-          //  wm = 180 - bounds.pminx + 180 + bounds.mmaxx;      // width of putative anti-meridian crossing area
-          //  wp = bounds.pmaxx0 - bounds.mminx0;                // width of putative meridian crossing area
-          //  // are these isolated ?
-          //  if (((bounds.mmaxx < -45) && (bounds.mminx0 > -45)) && ((bounds.pmaxx0 < 45) && (bounds.pminx > 45))) {
-          //    // open space between arbitrary +/-45 boundary
-          //    wx = wm - bounds.mmaxx + bounds.pmaxx0;         // width of map (?)
-          //    center_long = -0.5 * (wm - bounds.mmaxx) + 0.5 * (bounds.pmaxx0);
-          //  }
-          //}
-        }
-        else {            // not an overwide calculation  // bias toward eastern hemisphere removed in xgtlt()
-          if (wp == 0) {
-            wx = wm;
-          }         // override for single sided western hemisphere
-          if (wm == 0) {
-            wx = wp;
-          }         // override for single sided eastern hemisphere
-          if (xmaxp > 179 && xminm < -179) {      // Antimeridian span test
-            wx = wm + wp;                        // total width of eastern and western
-            if (wm > wp) {                       // determine wider group
-              center_long = xmm - wp / 2;        // adjust western mid/mean point by half width of eastern
-              if (center_long < -180) {
-                center_long = 360 - center_long;
-              }       // bias from -180
-            }                                    // e.g., USA
-            else {
-              center_long = xmp + wm / 2;        // adjust positive mean/mid point by half width of western
-              if (center_long > 180) {
-                center_long = center_long - 360;
-              }        // bias from +180
             }
-            // assume: calculate center about +/- 180 over contiguous spanning area
-            //center_long = (xminp + xmaxm) / 2.0;    // revised simplified calculation JRF 04AUG2015
-          }
-          else {                                 // i.e., if not Antimeridian span, center on extents about 0
-            // case disjoint areas divided by prime meridian or single point
-            if (wx != 0) {
-              center_long = (xminm + xmaxp) / 2;        // default calculation for both hemisperes having data
-              if (wp == 0) {
-                center_long = xmm;
-              }         // override for single sided western hemisphere
-              if (wm == 0) {
-                center_long = xmp;
-              }         // override for single sided eastern hemisphere
+            //if (bounds.anti || bounds.prime) {
```

Comment:
This is related to the deferred back-integration of anti-meridian handling into the maps code.